modules/racer/src/main/Env.scala

```diff
@@ -19,7 +19,7 @@ final class Env(
     db: lila.db.Db
 )(using Executor, Scheduler, play.api.Mode):
 
-  private lazy val colls = RacerColls(puzzle = puzzleColls.puzzle)
```

Comment:
This looks wrong, can We remove this line completely?

Reply:
sure